webapp/src/selectors/index.tsx

```diff
@@ -0,0 +1,16 @@
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+
+export const getServerRoute = (state) => {
```

Comment:
This is failing in the linter because you need to pass a type for the state, which can be `any`, but you have to provide that explicitly.

Reply:
@jespino My apologies I should have caught that. Pushed a fix :)